store/store.go

```diff
@@ -43,18 +47,18 @@ type BlockStore struct {
 	base   int64
 	height int64
 
-	ipfsDagAPI ipld.DAGService
+	dag format.DAGService
 }
 
 // NewBlockStore returns a new BlockStore with the given DB,
 // initialized to the last height that was committed to the DB.
-func NewBlockStore(db dbm.DB, dagAPI ipld.DAGService) *BlockStore {
+func NewBlockStore(db dbm.DB, dagAPI format.DAGService) *BlockStore {
```

Comment:
We need to ensure we only passing here Offline Dag

Reply:
Good idea, changed the API provider to have an online or offline option, and then started only using the offline version https://github.com/lazyledger/lazyledger-core/pull/374/commits/8619f275bd635f9d95891d755d3e9b700de514d6

Reply:
although, I couldn't find a way to quickly check inside the function that the `DAGService` is offline

Reply:
This is actually is not trivial and requires passing IPFS's blockstore and just DAG 😞

This is how the func for offline DAG would look like:
```go
func OfflineDAG(bstore blockstore.Blockstore) format.DAGService {
	return dag.NewDAGService(bsrv.New(bstore, offline.Exchange(bstore)))
}
```